schemas/expconf/v0/hyperparameter.json

```diff
@@ -24,13 +24,6 @@
                 "unionKey": "const:type=categorical",
                 "$ref": "http://determined.ai/schemas/expconf/v0/hyperparameter-categorical.json"
             },
-            {
```

Comment:
What was this for?

Reply:
Nested hyperparameters.  Once upon a time I thought the experiment config project was so quick and easy I'd just "throw nested hp in too".